<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifierEtudient</class>
 <widget class="QDialog" name="ModifierEtudient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier Etudient</string>
  </property>
  <widget class="QPushButton" name="MleN">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier le Nom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModifierCNE">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier CNE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModifierMassar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier code Massar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModifierDate">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier Date Naissance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModifierPrenom">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier Le Prenom</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
